lib/src/wrappers/ModalWrapper.tsx

```diff
@@ -63,21 +63,26 @@ export const ModalWrapper: React.FC<ModalWrapperProps<any>> = ({
   onSetToday,
   ...other
 }) => {
-  const handleKeyDown = React.useCallback(
-    (event: KeyboardEvent) => {
-      switch (event.key) {
-        case 'Enter':
-          onAccept();
-          break;
-        default:
-          return; // if key is not handled, stop execution
-      }
+  React.useEffect(() => {
```

Comment:
It would much clear to reuse created below `KeydownListener`

Reply:
Reusing will require to use useCallback. Now handler is builtin into effect.

Reply:
Why not to create useKeyDown for both cases?

Reply:
It has the same problem

Reply:
Okay, I will make it

Reply:
Done